fluent/StorageRune.ts

```diff
@@ -31,7 +31,7 @@ export class StorageRune<
       blockHash?: string,
     ]>
   ) {
-    const storageKey = this.$key.encoded(key).map(hex.encode)
+    const storageKey = this.$partialKey.encoded(key).map(hex.encode)
```

Comment:
This is a full key, not a partial key
```suggestion
    const storageKey = this.$key.encoded(key).map(hex.encode)
```